library/core/src/intrinsics/mod.rs

```diff
@@ -2786,6 +2786,7 @@ pub const fn carrying_mul_add<T: ~const fallback::CarryingMulAdd<Unsigned = U>,
 /// `x % y != 0` or `y == 0` or `x == T::MIN && y == -1`
 ///
 /// This intrinsic does not have a stable counterpart.
+#[rustc_intrinsic_const_stable_indirect]
```

Comment:
With this, you should be able to remove the `rustc_const_unstable` from the `slice_as_chunks` methods. (They will still remain unstable due to `#[unstable]`.)